Examples/Sample_xDscWebServiceRegistrationWithSecurityBestPractices.ps1

```diff
@@ -53,7 +53,7 @@ configuration Sample_xDscWebServiceRegistrationWithEnhancedSecurity
             DependsOn               = "[WindowsFeature]DSCServiceFeature" 
             RegistrationKeyPath     = "$env:PROGRAMFILES\WindowsPowerShell\DscService"   
             AcceptSelfSignedCertificates = $true
-            UseUpToDateSecuritySettings = $true
+            UseSecurityBestPractices = $true
```

Comment:
Let me add some details in the release note. Will send out another update after that.